samples/net/mqtt/boards/nrf7002dk_nrf5340_cpuapp.conf

```diff
@@ -36,6 +36,7 @@ CONFIG_NET_IPV4=y
 CONFIG_NET_TCP=y
 CONFIG_NET_UDP=y
 CONFIG_NET_SOCKETS_OFFLOAD=n
+CONFIG_NET_SOCKETS_POLL_MAX=4
```

Comment:
FYI, I am planning to do this by default in the WPA supplicant itself, but this should work fine.

Reply:
Removed this change.